templates/azure/private-offer.html

```diff
@@ -0,0 +1,63 @@
+{% extends "azure/base_azure.html" %}
+
+{% block title %}Ubuntu on Azure Private Offer{% endblock %}
```

Comment:
This comes out as "Ubuntu on Azure Private Offer | Ubuntu". I think we can use the repetition here by changing to "Azure Private Offer"

Reply:
Updaded